### PR TITLE
Rename Main nav menus and beautify People Page

### PR DESCRIPTION
**Pull request by KellyAH**
- https://github.com/zinc-collective/www.zinc.coop/issues/90
Clean up work discovered while working on ☝️ 

# Changes
- Rename main nav menus for clarity: 
  - change "About Us" ➡️ "Updates"
  - change "People" ➡️ "About Us"
- Clean up People Page
  - Add bio photo for zspencer (Alas I could not find any alternative photos, @zspencer please let me know if you want a different photo)
  - Capitalize and tweak people `type` and `link` values to be more accurate
  - Remove defunct 404 linked in URL for @zspencer 
  - Remove unflattering caret bullet points

changes

### CODE REVIEW

**Review by anaulin — COMMENTED** [review]
This looks fine to me, but I'll let @zspencer give the final approval, so that he can enthusiastically consent (or not) to his photo being added.

File: src/people/index.json [hunk 1]
```diff
@@ -17,11 +17,11 @@
       {
          "name": "Ana Ulin",
          "image": "/people/img/anaulin.jpg",
-         "type": "maintainer",
+         "type": "Maintainer",
```
Comment (zspencer) [inline src/people/index.json]: Ana is a Member now!

**Review by zspencer — APPROVED** [review]
I don't like having photos of my face on the Internet but I suppose I can until we can get an artist to draw something.

File: src/people/index.json (hunk 1 above)
Comment (KellyAH) [reply src/people/index.json]: fixed 27d8b0a594ee8d644292f560ae8e81e16f097313

**Comment by KellyAH** [comment]
> I don't like having photos of my face on the Internet but I suppose I can until we can get an artist to draw something.

@zspencer I have replaced your profile photo with a cute cartoon version. You are now adorable. 😉 
Screenshot 2023-06-09 at 5 30 17 PM

**Comment by KellyAH** [comment]
🚀 deployed and ✅ verified on prod 💃

**Comment by zspencer** [comment]
Thank you!!!

**Pull request merged**